### PR TITLE
Fix issues with building mscorlib inside VS

### LINKED ISSUE #3988: Issues when building mscorlib in VS
I attempted to build mscorlib (i.e. mscorlib.csproj) from Visual Studio but encountered several issues. None of them are fatal, since editing from VS and then building from the command line works, but they are annoying.

1. There is no solution file that contains mscorlib.csproj (not even after running build.cmd from the command line). This is slightly annoying by itself, but worse, when I directly open mscorlib.csproj in VS and attempt to build, I get the error:

 > Solution is not saved. Please save your solution before managing NuGet packages.

 Just saving the autogenerated single-project mscorlib.sln to disk fixes this.

2. When I try to build after that, I get this error:

 > error CS2007: Unrecognized option: '/test:moduleName=CommonLanguageRuntimeLibrary'

3. All .cs files are shown duplicated in Solution Explorer:

 Solution Explorer screenshot

### PR DESCRIPTION
**Pull request by AlexGhiondea**
Fixes #3988 

Will validate this in the TFS build before merging.

### CODE REVIEW

File: src/mscorlib/GenerateCompilerResponseFile.targets [hunk 1]
```diff
@@ -7,15 +7,8 @@
...
-    <WriteLinesToFile Condition="'$(UseRoslynCompiler)'!='true'"
-                      File="$(IntermediateOutputPath)\moduleName.rsp"
-                      Lines="/test:moduleName=CommonLanguageRuntimeLibrary"
-                      Overwrite="true" />
-
-    <!-- When Roslyn is enabled we need to set the runtimemetadataversion -->
-    <WriteLinesToFile Condition="'$(UseRoslynCompiler)'=='true'"
-                      File="$(IntermediateOutputPath)\moduleName.rsp"
+    <!-- We need to set the runtimemetadataversion -->
+    <WriteLinesToFile File="$(IntermediateOutputPath)\moduleName.rsp"
```
Comment (svick) [inline src/mscorlib/GenerateCompilerResponseFile.targets]: If `UseRoslynCompiler` is no longer necessary, should lines like `set UseRoslynCompiler=true` be removed from build.cmd/.sh etc?

Comment (AlexGhiondea) [inline src/mscorlib/GenerateCompilerResponseFile.targets]: Most likely yes. I am checking to see if this is used anymore in the internal build.

Comment (AlexGhiondea) [inline src/mscorlib/GenerateCompilerResponseFile.targets]: I think we should be ok to remove those from the build. Could you file an issue to remove those?

**Comment by AlexGhiondea** [comment]
@jkotas, @tarekgh could you take a look?

**Comment by jkotas** [comment]
LGTM

**Comment by tarekgh** [comment]
LGTM too.

File: src/mscorlib/GenerateCompilerResponseFile.targets (hunk 1 above)
Comment (svick) [inline src/mscorlib/GenerateCompilerResponseFile.targets]: Done: #4032.

**Pull request merged**